src/coreclr/jit/lowerxarch.cpp

```diff
@@ -3429,7 +3429,7 @@ void Lowering::LowerHWIntrinsicDot(GenTreeHWIntrinsic* node)
                     //   e6, e7, e4, e5 | e2, e3, e0, e1
                     //   e7, e6, e5, e4 | e3, e2, e1, e0
 
-                    shuffleConst = 0x4D;
+                    shuffleConst = 0x4E;
```

Comment:
I ran tests on all ISAs from SSE2-AVX2 to ensure no other "disabled" instructions were being generated and it flagged this as a failing test (`4D` meant it was selecting `1, 3, 0, 1`. `4E` means it will pick `2, 3, 0, 1`)